public/_redirects

```diff
@@ -1 +0,0 @@
-/*    /index.html   200
```

Comment:
@saifulkhan I removed this file as it does not seem to be used for anything. Am I correct in this assumption or shall I revert the change?